android/src/ch/opengis/qfield/QFieldActivity.java

```diff
@@ -98,6 +98,7 @@ private void prepareQtActivity() {
         new File(qFieldDir + "basemaps/").mkdir();
         new File(qFieldDir + "fonts/").mkdir();
         new File(qFieldDir + "proj/").mkdir();
+        new File(qFieldDir + "authentications/").mkdir();
```

Comment:
```suggestion
        new File(qFieldDir + "auth/").mkdir();
```

what about making this shorter?

Reply:
No strong feeling, we can go with `auth`.

Reply:
@m-kuhn , done.